homeassistant/components/calendar/__init__.py

```diff
@@ -141,15 +162,7 @@ def state_attributes(self):
         if event is None:
             return None
 
-        event = normalize_event(event)
-        return {
-            "message": event["message"],
-            "all_day": event["all_day"],
-            "start_time": event["start"],
-            "end_time": event["end"],
-            "location": event["location"],
-            "description": event["description"],
-        }
+        return get_normalized_event(event)
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

Reply:
Oh that's odd as I have another PR [47049](https://github.com/home-assistant/core/pull/47049) which implements similar changes and hasn't been flagged for this.

Reply:
No one has looked at that one yet.